src/index.html

```diff
@@ -70,8 +70,13 @@ <h3>Datenschutz</h3>
       <h3>Settings</h3>
       <h4>Legende</h4>
       <form class="settings-form">
+        <label for="labelScheme">Farbschema</label><br />
         <input class="setting" type="radio" name="labelScheme" value="RKI" /> RKI (Einfach) <br />
         <input class="setting" type="radio" name="labelScheme" value="RiskLayer" /> RiskLayer (Feingranular) <br />
+        <br />
+        <label for="interpolate">Farbinterpolation</label><br />
+        <input class="setting" type="radio" name="interpolate" value="None" /> Keine <br />
+        <input class="setting" type="radio" name="interpolate" value="Linear" /> Lineare Interpolation <br />
```

Comment:
Maybe it makes sense to add a disclaimer here that this is meant for entertainment/artistic purposes and no guarantees for correctness? Or does the general statement in the top right of the main page suffice?